src/Umbraco.Web.Common/Security/MembersUserManager.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections.Generic;
+using System.Security.Principal;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Umbraco.Core;
+using Umbraco.Core.Configuration.Models;
+using Umbraco.Core.Models.Membership;
+using Umbraco.Core.Security;
+using Umbraco.Extensions;
+using Umbraco.Infrastructure.Security;
+using Umbraco.Net;
+using Umbraco.Web.Models.ContentEditing;
+
+
+namespace Umbraco.Web.Common.Security
+{
+    public class MembersUserManager : UmbracoUserManager<MembersIdentityUser, MemberPasswordConfigurationSettings>, IMembersUserManager
+    {
+        private readonly IHttpContextAccessor _httpContextAccessor;
+
+        public MembersUserManager(
+            IIpResolver ipResolver,
+            IUserStore<MembersIdentityUser> store,
+            IOptions<MembersIdentityOptions> optionsAccessor,
+            IPasswordHasher<MembersIdentityUser> passwordHasher,
+            IEnumerable<IUserValidator<MembersIdentityUser>> userValidators,
+            IEnumerable<IPasswordValidator<MembersIdentityUser>> passwordValidators,
+            BackOfficeIdentityErrorDescriber errors,
+            IServiceProvider services,
+            IHttpContextAccessor httpContextAccessor,
+            ILogger<UserManager<MembersIdentityUser>> logger,
+            IOptions<MemberPasswordConfigurationSettings> passwordConfiguration)
+            : base(ipResolver, store, optionsAccessor, passwordHasher, userValidators, passwordValidators, errors, services, logger, passwordConfiguration)
+        {
+            _httpContextAccessor = httpContextAccessor;
+        }
+
+        private string GetCurrentUserId(IPrincipal currentUser)
```

Comment:
I think we should remove this stuff along with all of the events and have a separate task to see if we want or need any of it. For now lets keep it as simple as possible. We didn't have events like this before for members.

Reply:
Have removed all events except the ones required, and left comment.